example-config/ExampleSolarizedColourExtension.hs

```diff
@@ -38,13 +53,14 @@ solarizedDark =
   defaultColourConfig
     -- Set the default foreground colour of text of the terminal.
     { foregroundColour = Set (createColour 131 148 150) -- base0
+    , foregroundColour = Set (createColour   0  43  54) -- base03
```

Comment:
It looks like the tests are failing because of the duplicate field here (`foregroundColour`), and below as well.

Could you fix that up?

After you do that, I'll be happy to merge this in.

Reply:
Sure can.

Apologies for missing that in the transfer from my production config to the example.